tools/build/scripts/utils/addPackageMetadata.js

```diff
@@ -15,8 +15,14 @@ const rootPackageJson = process.env.RUSHSTACK_FILE_ERROR_BASE_FOLDER
 // Check if path to root package.json is valid.
 const rootPackageJsonPath = require.resolve(rootPackageJson);
 
+//Find the package.json of the project and retrieve the version and repository URL
+const packageJsonPath = path.join(process.cwd(), "package.json");
+const packageJson = JSON.parse(FS.readFileSync(packageJsonPath));
+const version = packageJson.version;
+const repositoryUrl = packageJson.repository.url;
```

Comment:
might want to add a fallback; repos should have these but just in case they dont